packages/preferences-controller/CHANGELOG.md

```diff
@@ -7,14 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [19.0.0]
+
 ### Changed
 
+- **BREAKING:** Bump peer dependency `@metamask/keyring-controller` from `^22.1.1` to `^23.0.0` ([#6345](https://github.com/MetaMask/core/pull/6345))
 - Bump `@metamask/base-controller` from `^8.0.1` to `^8.1.0` ([#6284](https://github.com/MetaMask/core/pull/6284))
 - Bump `@metamask/controller-utils` from `^11.11.0` to `^11.12.0` ([#6303](https://github.com/MetaMask/core/pull/6303))
 
-### Deprecated
+### Removed
 
-- Deprecate preference `smartAccountOptInForAccounts` and function `setSmartAccountOptInForAccounts` ([#6087](https://github.com/MetaMask/core/pull/6087))
+- **BREAKING:** Remove `smartAccountOptInForAccounts` from `PreferencesState` and `setSmartAccountOptInForAccounts` method ([#6079](https://github.com/MetaMask/core/pull/6079))
```

Comment:
Nit: this change can be rolled back as we actually reverted the breaking change for removal of `smartAccountOptInForAccounts` we are only marking it as `deprecated`.